pom.xml

```diff
@@ -2313,7 +2313,7 @@
     <properties>
 
         <!--Carbon Identity Framework Version-->
-        <carbon.identity.framework.version>5.25.541</carbon.identity.framework.version>
+        <carbon.identity.framework.version>5.25.542</carbon.identity.framework.version>
```

Comment:
The PR builder for this version bump was passed https://github.com/wso2/product-is/pull/18000#pullrequestreview-1741522920